### PR TITLE
create evoong.md

### PR DESCRIPTION
**Pull request by evoong**
Fixes #issue_number

### Description
Description of changes made.

### Raw.Githack preview link
raw.githack link to page(s) changed.

https://raw.githack.com/evoong/evoong.github.io/new_branch/#!pages/vi/profiles/evoong.md

### CODE REVIEW

File: pages/vi/profiles/evoong.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- **Time Zone:** EST
+- **OS:** Windows 10 Home Version 1909
+
+## Description:
+⋅⋅* I am a second year Nanotechnology Student from the Unviersity of Waterloo.
+⋅⋅* It is a pleasure meet you! I like solving, designing, and creating with a passion. 
+⋅⋅* I am someone who gets absorbed into my projects as time passes by and loves exceeding expectations. 
+⋅⋅* My goal is to work on a service or product that can make a change in a persons everyday life and have an impact on the world. 
+
+![My Linkedin](https://www.linkedin.com/in/eric-voong/)
```
Comment (kjong) [inline pages/vi/profiles/evoong.md]: You should change this markdown element to a link, not an image.

**Review by kjong — CHANGES_REQUESTED** [review]

File: pages/vi/profiles/evoong.md (hunk 1 above)
Comment (evoong) [reply pages/vi/profiles/evoong.md]: done

**Review by evoong — COMMENTED** [review]
I finished making the changes

**Review by LordJashin32 — APPROVED** [review]

**Comment by dogi** [comment]
duplicate

**Pull request closed without merging**